pulsar-client/src/main/java/org/apache/pulsar/client/impl/ConsumerBuilderImpl.java

```diff
@@ -327,6 +327,14 @@ public ConsumerBuilder<T> acknowledgmentGroupTime(long delay, TimeUnit unit) {
         return this;
     }
 
+    @Override
+    public ConsumerBuilder<T> acknowledgmentGroupSize(int messageNum) {
+        checkArgument(messageNum >= 0, "acknowledgementsGroupSize needs to be >= 0");
```

Comment:
Maybe we need to make it greater than 0?

Reply:
Fixed,PTAL,thanks! @RobertIndie